CRM/Civicase/Hook/ValidateForm/SaveActivityDraft.php

```diff
@@ -66,7 +66,7 @@ private function saveDraftForSpecialForms($formName, CRM_Core_Form &$form, array
       'status_id' => 'Draft',
       'case_id' => $caseId,
       'id' => $form->getVar('_activityId'),
-      'assignee_id' => $form->getVar('_contactIds'),
+      'assignee_id' => CRM_Core_Session::getLoggedInContactID(),
```

Comment:
Just thinking more about this and I think we should add these form contacts (i.e contacts in the TO field) as targets since the emails are meant for them and we can record this activity for them.

So we will have something like:
```php
'target_id' => $form->getVar('_contactIds'),
```
What do you think?

Reply:
@tunbola that's exactly the code we had before. The problem with adding the form contacts is that the email is meant for them, but the one who is preparing the email is the one assigned to it because the email has not been sent yet. So the contact needs to be assigned the email so they finish the draft and send the email.

Reply:
I see @reneolivo . The code I pasted is different from the one before though?
`assignee_id` => `target_id`
But thinking more about it now, since it's draft, the activity should be kept with creator, so code is fine as is.